### DISCUSSION TITLE
Language files detected ok - but not coloured

### DISCUSSION THREAD

**Discussion by GordonSmith** [github-linguist/linguist#5499] [Q&A] [answered]
Where do I begin to debug a no colourisation issue.

The language (ECL) has been detected fine, but I am not getting any colourisation when I view a file (the colourisation was only added in the latest version so expect its a me issue - just not sure where to start)...

Example: 
* https://github.com/hpcc-systems/vscode-ecl/search?l=ecl
* https://github.com/hpcc-systems/vscode-ecl/blob/5653c853f905821718535ae20b6a6ed684df0629/ecl-sample/ProgGuide/BadWordSearch.ecl

**Comment by lildude** [comment, marked as answer]
🤦 I completely missed that your PR didn't include an update to the `languages.yml` file (and the `vendor/README.md` but that's not important) so the `tm_scope` is still set to `none` and thus your grammar isn't being used:

https://github.com/github/linguist/blob/493007b50417ec4f655afe57ba7044b4b929f1cf/lib/linguist/languages.yml#L1408-L1418

I'll open a PR to make the change and try and get a patch release out today or early next week.

**Reply by lildude** [reply]
PR https://github.com/github/linguist/pull/5500

**Reply by GordonSmith** [reply]
@lildude - Thanks for your prompt response BTW, its appreciated (edit:  just saw your comment re the patch).

**Reply by lildude** [reply]
You'll need to wait unfortunately.

**Reply by lildude** [reply]
Should all be nice and pretty and colourful now. You might need to perform a forced browser refresh to clear its cache.

**Reply by GordonSmith** [reply]
Saw that (and had done sanity check in lightshow) - thanks for all your help with this!
